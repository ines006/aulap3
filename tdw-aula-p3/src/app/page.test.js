import React from 'react';
import { render } from '@testing-library/react';
import Home from './page';


describe('Home component', () => {

    const punycode = require('punycode');

    test('renders Home correctly', () => {
      const { asFragment } = render(<Home />);
      expect(asFragment()).toMatchSnapshot();
    });
  });