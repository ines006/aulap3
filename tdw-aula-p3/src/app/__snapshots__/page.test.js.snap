// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders Home correctly 1`] = `
<DocumentFragment>
  <div
    class="page"
  >
    <main
      class="main"
    >
      <img
        alt="Next.js logo"
        class="logo"
        data-nimg="1"
        decoding="async"
        fetchpriority="high"
        height="38"
        src="https://nextjs.org/icons/next.svg"
        style="color: transparent;"
        width="180"
      />
      <div>
        <h1>
          Welcome to the Home page!
        </h1>
        <p>
          This is the home page of our application.
        </p>
      </div>
      <ol>
        <li>
           Teste de prettier
        </li>
         
        <li>
          mais um teste 
        </li>
         
        <li>
           Teste de prettier
        </li>
         
        <li>
          maais um teste 
        </li>
         
        <li>
           Teste de prettier
        </li>
         
        <li>
          maais um teste 
        </li>
         
        <li>
           Teste de prettier
        </li>
         
        <li>
          maais um teste 
        </li>
        <li>
           alterar page home para snapchot
        </li>
        <li>
           alterar page home para snapchot 2
        </li>
      </ol>
      <div
        class="ctas"
      >
        <a
          class="primary"
          href="https://vercel.com/new?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="Vercel logomark"
            class="logo"
            data-nimg="1"
            decoding="async"
            height="20"
            loading="lazy"
            src="https://nextjs.org/icons/vercel.svg"
            style="color: transparent;"
            width="20"
          />
          Deploy now
        </a>
        <a
          class="secondary"
          href="https://nextjs.org/docs?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
          rel="noopener noreferrer"
          target="_blank"
        >
          Read our docs
        </a>
      </div>
    </main>
    <footer
      class="footer"
    >
      <a
        href="https://nextjs.org/learn?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="File icon"
          aria-hidden="true"
          data-nimg="1"
          decoding="async"
          height="16"
          loading="lazy"
          src="https://nextjs.org/icons/file.svg"
          style="color: transparent;"
          width="16"
        />
        Learn
      </a>
      <a
        href="https://vercel.com/templates?framework=next.js&utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Window icon"
          aria-hidden="true"
          data-nimg="1"
          decoding="async"
          height="16"
          loading="lazy"
          src="https://nextjs.org/icons/window.svg"
          style="color: transparent;"
          width="16"
        />
        Examples
      </a>
      <a
        href="https://nextjs.org?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Globe icon"
          aria-hidden="true"
          data-nimg="1"
          decoding="async"
          height="16"
          loading="lazy"
          src="https://nextjs.org/icons/globe.svg"
          style="color: transparent;"
          width="16"
        />
        Go to nextjs.org →
      </a>
    </footer>
  </div>
</DocumentFragment>
`;

exports[`renders Home correctly 1`] = `
<div>
  <ol>
    <li>
      alterar page home para snapchot
    </li>
  </ol>
  <div
    class="ctas"
  >
  </div>
</div>
`;
